slack-java-client/src/main/java/com/hubspot/slack/client/SlackWebClient.java

```diff
@@ -897,8 +898,7 @@ protected LazyLoadingPage<Result<List<Conversation>, SlackError>, String> getPag
         );
 
         CompletableFuture<Result<List<Conversation>, SlackError>> pageFuture = resultFuture.thenApply(
-          result -> result.mapOk(ConversationListResponse::getConversations)
-        );
+          result -> result.mapOk(response -> getConversations(response, params)));
```

Comment:
1. We don't really need to pass the whole `params` object to this method. 
2. We can store `params.shouldExcludeSharedChannels().orElse(false)` in a variable once even before we create `AbstractPagedIterable` and pass it to `getConversations`.